File: JAVA/2678. Number of Senior Citizens.java
class Solution {
    public int countSeniors(String[] details) {
        int c = 0;
        for(int i=0;i<details.length;++i)
        {
            int x = details[i].charAt(11) - '0';
            int y = details[i].charAt(12) - '0';
            if((x*10 + y) > 60)
                c++;
        }
        return c;
    }
}

File: C++/2678. Number of Senior Citizens.cpp
class Solution {
public:
    int countSeniors(vector<string>& details) {
        int SeniorCtz=0;
        for(string &detail:details){
            int onesDigit=detail[12]-'0';
            int tensDigit=detail[11]-'0';
            int age=tensDigit*10+onesDigit;
            if(age>60){
                SeniorCtz++;
            }        
        }
        return SeniorCtz;
    }
};
